### ISSUE TITLE
namespace issues

### ISSUE THREAD

**Issue by tuckerdotcodes** [MeadowSuite/Meadow#11]
I'm a novice with this extension, so it may be an issue between the chair and keyboard. I have attached a print screen of my error message.
![capture](https://user-images.githubusercontent.com/18519239/49837325-33f3b780-fd63-11e8-9c9e-bc5d5da66bd1.PNG)

**Comment by Xenomega** [comment]
Hey @tuckerdotcodes ,

I see the folder structure on the left doesn't have any "GeneratedContracts" folder. If I had to take a guess here, you probably dropped your smart contracts in and tried to reference the C# interfaces that are generated for them immediately, before you actually generated them. You actually need to build the project to generate the C# interfaces, then you can reference them. The C# interface generation is done through MSBuild.

For more information please check out the following link on help getting started (specifically, the "Build project" heading): [Getting Started](https://github.com/MeadowSuite/Meadow/wiki/Getting-Started-with-Unit-Tests)

tl;dr: C# interfaces for the smart contracts are generated on build. It seems you haven't built your project since you put your smart contracts in.

Side note: You can also use full Visual Studio and other C# supporting IDEs with Meadow if you prefer. VSCode is only required for the Solidity debugger. I know some users would rather avoid command-line usage which VSCode forces you to use at times. Those users may prefer using full Visual Studio enterprise on Windows if they are just running unit tests or etc.
